spdmlib/src/message/version.rs

```diff
@@ -106,18 +106,18 @@ impl SpdmCodec for SpdmVersionResponsePayload {
             },
             MAX_SPDM_VERSION_COUNT,
         );
+        let mut version_exist_map: [u8; MAX_SPDM_VERSION_COUNT] = [0; MAX_SPDM_VERSION_COUNT];
```

Comment:
We need to count the version count number from Reader.
Thus, we need to figure out which version already parsed. 
Unfortunately, default value is same as a legal SpdmVersion10 entry, we cannot figure out whether to count a 1.0 version entry.
As the result, a map is used to indicate whether a version has been counted.